workers/social/lib/social/traits/notifiable.coffee

```diff
@@ -33,6 +33,25 @@ module.exports = class Notifiable
           account.sendNotification 'InstanceChanged', opts
 
 
+  updateTemplateAndNotify: (options, change, callback) ->
```

Comment:
traits shouldn't include something specific to a model, they are generic. we can move this into `JStackTemplate` model instead.